diff.py

```diff
@@ -1412,6 +1521,57 @@ def _normalize_bl(self, mnemonic: str, row: str) -> str:
         row, _ = split_off_address(row)
         return row + "<ignore>"
 
+    def _normalize_data_pool(self, row: str) -> str:
+        pool_match = re.search(ARM32_LOAD_POOL_PATTERN, row)
+        if pool_match:
+            offset = pool_match.group(3).split(" ")[0][1:]
+            repl = DATA_POOL_PLACEHOLDER.format(offset)
+            return pool_match.group(1) + repl
+        return row
+
+
+class PostProcessor:
+    def __init__(self, config: Config) -> None:
+        self.config = config
+
+    def post_process(self, lines: List["Line"]) -> None:
+        return
+
+
+class PostProcessorARM32(PostProcessor):
+    def __init__(self, config: Config) -> None:
+        super().__init__(config)
+        self.lines_by_line_number = {}
+
+    def post_process(self, lines: List["Line"]) -> None:
+        self.lines_by_line_number = self.extract_lines_by_line_number_(lines)
+        for line in lines:
+            reloc_match = re.search(
+                DATA_POOL_PLACEHOLDER_PATTERN, line.normalized_original
+            )
+            if reloc_match is None:
+                continue
+
+            # Get value at relocation
+            reloc = reloc_match.group(0)
+            line_number = re.search(DATA_POOL_PLACEHOLDER_PATTERN, reloc).group(1)
+            line_original = self.lines_by_line_number[int(line_number, 16)].original
+            value = line_original.split()[1]
+
+            # Replace relocation placeholder with value
+            replaced = re.sub(
+                DATA_POOL_PLACEHOLDER_PATTERN,
+                f"={value} ({line_number})",
+                line.normalized_original,
+            )
+            line.original = replaced
+
+    def extract_lines_by_line_number_(self, lines: List["Line"]) -> Dict[int, "Line"]:
```

Comment:
not worth a separate function for this when it's 3 lines of code and used from only one place; inline it

Reply:
Done.